File: CRS-Signify-SpringRest-Group-G/src/main/java/com/signify/bean/Course.java
package com.signify.bean;

public class Course {
	private String courseCode, name, instructor;
	private int numStudents, type, semester;

	public int getSemester() {
		return semester;
	}

	public void setSemester(int semester) {
		this.semester = semester;
	}

	private double fee;

	public double getFee() {
		return fee;
	}

	public void setFee(double fee) {
		this.fee = fee;
	}

	public int getType() {
		return type;
	}

	public void setType(int type) {
		this.type = type;
	}

	public int getNumStudents() {
		return numStudents;
	}

	public void setNumStudents(int numStudents) {
		this.numStudents = numStudents;
	}

	public String getCourseCode() {
		return courseCode;
	}

	public void setCourseCode(String courseCode) {
		this.courseCode = courseCode;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getInstructor() {
		return instructor;
	}

	public void setInstructor(String instructor) {
		this.instructor = instructor;
	}

}

File: CRS-Signify-UI-UX-Group-G/src/app/model/course.ts
export class Course {
    courseCode: string;
    name: string;
    instructor: string | null;
    numStudents: number;
    type: number;
    semester: number;
    fee: number;

    constructor(courseCode: string, name: string, instructor: string, numStudents: number, type: number, semester: number, fee: number) {
        this.courseCode = courseCode;
        this.name = name;
        this.instructor = null;
        this.numStudents = numStudents;
        this.type = type;
        this.semester = semester;
        this.fee = fee;
    }
}
